backend/app/tests/crud/test_generic_crud.py

```
@@ -0,0 +1,169 @@
+    result = crud.get_by_id(session=mock_db_session, model=MockDbModel, id=obj_id)
+    mock_db_session.get.assert_called_once_with(MockDbModel, obj_id)
+    assert result == mock_instance
+
+def test_generic_get_by_id_not_found(mock_db_session: MagicMock):
+    obj_id = uuid.uuid4()
+    mock_db_session.get.return_value = None
+    result = crud.get_by_id(session=mock_db_session, model=MockDbModel, id=obj_id)
+    mock_db_session.get.assert_called_once_with(MockDbModel, obj_id)
+    assert result is None
+
+@patch("app.crud.select") 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)